### ISSUE TITLE
Transparency of rod background color is applied to rod color in bar chart.

### ISSUE THREAD

**Issue by drekka** [imaNNeoFighT/fl_chart#575]
Hi, I'm attempting to use a semi-transparent rod background color to create a subtle bar background an so the horizontal lines show through. But what I'm finding is that the alpha of the rod background is overriding the alpha of the rod, so it becomes transparent as well.

I was expecting to be able to have an opaque rod with a semi-transparent background. Is this is bug or am I missing something?

**Comment by imaNNeoFighT** [comment]
Hi there.
I checked and it is working correctly.

Open bar_chart_sample1.dart.
change line 115 to `colors: [Colors.red],`.
change line 120 to `colors: [Colors.red.withOpacity(0.2)],`

And this is what you get:

![image](https://user-images.githubusercontent.com/7009300/108162117-b164a080-7101-11eb-8aa0-9b48e9b9a414.png)

**Comment by drekka** [comment]
hmmm. Ok. let me try again.

**Comment by drekka** [comment]
Ok, it's something to do with gradients. I changes my code to use `Colors.red` like your example and it worked as you sia.d But when I changed it too `[Colors.red, Colors.yellow]` it went transparent like this:

![Screen Shot 2021-02-18 at 3 00 44 pm](https://user-images.githubusercontent.com/153837/108303496-20bdbb80-71fa-11eb-9659-0e6b1513314b.png)

This is probably something to do with my poor understanding of colors in material design or something (it hurts my head trying to follow the docs).
